### ISSUE TITLE
Develop COG for more Information about Sponsorships Deals

### ISSUE THREAD

**Issue by natesinger** [Shift-Cyber/bits#25]
Cog to give user the ability to view details about current sponsor deals. Need to figure out how we want to do content management and also how we want to deploy it. I think it makes sense to do this within the admin panel once we issue version 2 of the bot, but until its tied to the web app we have to do content management some other way. I also want to make sure this is somewhat clean, not oddly cluttered... maybe deleting the message after it recognizes the command, not sure. This would probably be part of the 1.1 release for the time being. 

Long story short, if we have a sponsor deal like 30% off at NoStarch, how can we give them commands to find out about it?

**Comment by natesinger** [comment]
Cherrypicked for Bits Release v1.1.0: Minor
